### PR TITLE
add new setAdmins

### PR DESCRIPTION
**Pull request by Bodasyuk**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by Liubov-H — APPROVED** [review]

**Review by IlliaDR — APPROVED** [review]

File: src/pages/City/AddAdministratorModal/AddCitiesSecretaryForm.tsx [hunk 1]
```diff
@@ -83,7 +83,7 @@ const AddCitiesNewSecretaryForm = (props: any) => {
   }, [props]);
 
   return (
-    <Form name="basic" onFinish={handleSubmit} form={form} className="formAddSecretaryModal">
+    <Form name="basic" onFinish={(values) => {handleSubmit(values); setLoading(true)}} form={form} className="formAddSecretaryModal">
```
Comment (andrii52) [inline src/pages/City/AddAdministratorModal/AddCitiesSecretaryForm.tsx:86]: Where do you use `setLoading(false)` after this action is done?

File: src/pages/City/City/City.tsx [hunk 2]
```diff
@@ -466,6 +466,8 @@ const City = () => {
           const check = await getCheckPlastMember(admin.userId);
           if(check.data){
             await addCityAdmin(admin);
+            admins.push(admin);
```
Comment (andrii52) [inline src/pages/City/City/City.tsx:469]: What if `addCityAdmin` fails? You will still display that a new admin was added. I suggest fetching admins from server, or at least use `.then()`.

File: src/pages/City/AddAdministratorModal/AddCitiesSecretaryForm.tsx (hunk 1 above)
Comment (Bodasyuk) [reply src/pages/City/AddAdministratorModal/AddCitiesSecretaryForm.tsx:86]: in button

Comment (Bodasyuk) [reply src/pages/City/AddAdministratorModal/AddCitiesSecretaryForm.tsx:86]: it sets button disabled(in loading)

File: src/pages/City/City/City.tsx (hunk 2 above)
Comment (Bodasyuk) [reply src/pages/City/City/City.tsx:469]: ok, ty

**Pull request merged**